changelog_unreleased/javascript/13532.md

```diff
@@ -0,0 +1,26 @@
+#### Fix unstable template literals with embedded languages (#13532 by @thorn0)
+
+If a template literal with embedded syntax is the only argument of a call or the body of an arrow function, it won't be printed on a new line unless its first line doesn't fit on the current line.
+
+<!-- prettier-ignore -->
+```js
+// Input
+foo(/* HTML */ `<!-- bar1 --> bar <!-- bar2 -->`);
+
+// Prettier stable (first output)
+foo(
+  /* HTML */ `<!-- bar1 -->
+    bar
+    <!-- bar2 -->`
+);
+
+// Prettier stable (second output)
+foo(/* HTML */ `<!-- bar1 -->
+  bar
+  <!-- bar2 -->`);
+
+// Prettier main (first output)
+foo(/* HTML */ `<!-- bar1 -->
+  bar
+  <!-- bar2 -->`);
```

Comment:
For this specific case hugging actually doesn't look good to me.
Probably, it'd make sense to distinguish between these two cases and do hugging only for the 2nd:

```js
// 1
foo(
  /* HTML */ `<!-- bar1 -->
    bar
    <!-- bar2 -->`
);

// 2
foo(/* HTML */ `
  <!-- bar1 -->
  bar
  <!-- bar2 -->
`);
``` 

Thoughts on whether this makes sense and on how to implement this?